kafka-connect-neo4j/src/main/kotlin/streams/kafka/connect/sink/Neo4jSinkConnectorConfig.kt

```diff
@@ -55,7 +56,10 @@ class Neo4jSinkConnectorConfig(originals: Map<*, *>): AbstractConfig(config(), o
     val batchTimeout: Long
     val batchSize: Int
 
-    val topicMap: Map<String, String>
+    val cdcTopics: Map<TopicType, Set<String>>
```

Comment:
data class Topics(cypherTopics, sourceIdTopics, schemaTopics)

move the validation code
and getXXX methods into that data class

use the class both from the sink plugin and the extension to save the duplicated code.